### DISCUSSION TITLE
Remove bad links from Wiki

### DISCUSSION THREAD

**Discussion by TGNYC** [SelfControlApp/selfcontrol#863] [General]
There are some bad links that redirect to spam / malicious websites in the wiki, and I wanted to highlight that here. For example, on the FAQ: http://work.simonremiszewski.com/index.php/project/sabbath/ is now a SPAM website.

**Comment by slambert** [comment]
Done! Thanks.

**Comment by slambert** [comment]
If you found others, let me know.
